src/modules/generateRandomNumber.js

```diff
@@ -7,7 +7,23 @@
  * @return {number} A random 4-digit number
  */
 function generateRandomNumber() {
-  /* Write your code here */
+  const numbers = [];
+  const [start, end] = [1, 9];
```

Comment:
The range `[1, 9]` ensures that 0 is never included in the generated number. Consider changing the range to `[0, 9]` and ensuring that the first digit is not 0 by swapping it with a non-zero digit if necessary.